controllers/clusterversion_controller.go

```diff
@@ -58,7 +60,18 @@ func (r *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	if err := r.Client.Get(context.TODO(), req.NamespacedName, &instance); err != nil {
 		return ctrl.Result{}, err
 	}
-	if err := r.ensureConsolePlugin(instance.Status.Desired.Version); err != nil {
+
+	if err := r.ensureOdfConsoleConfigMapAndService(); err != nil {
+		logger.Error(err, "Could not ensure configmap and service for odf-console deployment")
+		return ctrl.Result{}, err
+	}
+
+	if err := util.EnsureCSVsPresent(ctx, r.Client, r.OperatorNamespace, EssentialCSVs...); err != nil {
+		logger.Error(err, "Could not ensure CSVs in Succeeded state")
+		return ctrl.Result{Requeue: true, RequeueAfter: time.Second * 2}, nil
```

Comment:
2 sec is demanding, the observed time was 60s in total and 10s interval will be relaxing. (or) remove the time and add a csv predicate creation event w/ essential CSVs.

Reply:
I don't want to add long intervals, Adding a longer interval can delay the creation of the web console, which means a delay in the installation. Also, this will happen only when there is an installation or upgrade. It reminds me to handle the upgrade case.

Reply:
> I don't want to add long intervals,
- not sure for what qualifies as longer or shorter interval in a reconciliation and so dropping my comment.